dev-client/src/screens/CreateProjectScreen/components/CreateProjectForm.tsx

```diff
@@ -31,11 +31,8 @@ import ProjectForm, {
 } from 'terraso-mobile-client/screens/CreateProjectScreen/components/ProjectForm';
 import {useDispatch} from 'terraso-mobile-client/store';
 
-type Props = {
-  onInfoPress: () => void;
-};
-
-export const CreateProjectForm = ({onInfoPress}: Props) => {
+type Props = {};
+export const CreateProjectForm = ({}: Props) => {
```

Comment:
Suggestion: You could just remove this, right? 
```suggestion
export const CreateProjectForm = () => {
```

Reply:
We can yeah.